### ISSUE TITLE
conversations containing only one message empty

### ISSUE THREAD

**Issue by GoogleCodeExporter** [satyajit319/websmsdroid#309]
```
What steps will reproduce the problem?

1. open SMdroid
2. select one of the conversations containing only one message "(1)"


What is the expected output? What do you see instead?

After step 2. you'll see only "reply" but no mesage content.


What version of the product are you using? On what operating system?

donated v0.2.1 german @HTC Hero GSM running HTC's android 2.1


Please provide any additional information below.

No matter it it was an incoming or outgoing message.
```

Original issue reported on code.google.com by `ronnysef...@gmail.com` on 26 Aug 2010 at 9:19
- Merged into: #303

**Comment by GoogleCodeExporter** [comment]
Original comment by `felix.bechstein` on 26 Aug 2010 at 5:30
- Changed state: **Duplicate**
